tutor-contrib-harmony-plugin/tutor_k8s_harmony_plugin/patches/kustomization

```diff
@@ -19,6 +19,7 @@ patches:
 - path: plugins/k8s_harmony/k8s/deployment-revision-history.yaml
   target:
     kind: Deployment
+{%- if K8S_HARMONY_ENABLE_SHARED_SEARCH_CLUSTER %}
```

Comment:
Is this related?

Reply:
It just fixes an issue with when ES is disabled. Another PR was open with the fix. Will rebase once is merged